src/portable/raspberrypi/rp2040/rp2040_usb.h

```diff
@@ -80,7 +80,7 @@ struct hw_endpoint
     uint8_t *user_buf;
 
     // Data needed from EP descriptor
-    uint wMaxPacketSize;
+    uint16_t wMaxPacketSize;
```

Comment:
although this is correct, this structure is also used by another usb stack (one implemented the BOOTROM). TinyUSB is not the only stack that work with rp2040 mcu, therefore it still would be `uint` for compatible. I am fine with this though.